server/meshmodel/argo-cd/relationships/edge_permission.json

```diff
@@ -0,0 +1,43 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Edge",
+  "metadata": {
+    "description": "A relationship that defines permission for components, It ensures that only authorized components can interact with each other."
```

Comment:
@aniskhalfallah, please offer a description that is specific to this set of relationships and their meaning.

Reply:
Checking in...